include/grpcpp/completion_queue.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright 2015 gRPC authors.
+ * Copyright 2019 gRPC authors.
```

Comment:
That's a bit weird for a change :-)

Reply:
Thanks for catching this, not sure how it showed up. Cleaned up now.